src/Pretzel.Logic/Minification/JsMinifier.cs

```diff
@@ -20,13 +21,16 @@ public JsMinifier(IFileSystem fileSystem, IEnumerable<FileInfo> files, string ou
 
         public void Minify()
         {
-            var minifer = new Minifier();
-            var codeSettings = new CodeSettings();
-
             var content = fileSystem.BundleFiles(files);
-            var minified =  minifer.MinifyJavaScript(content, codeSettings);
-
-            fileSystem.File.WriteAllText(outputPath, minified);
+            var minified = Uglify.Js(content);
+            if(minified.HasErrors)
+            {
+                foreach(var error in minified.Errors)
+                {
+                    Console.WriteLine(error.ToString());
```

Comment:
Should be `Tracing.Error` or Warning depending on what is in `Errors`.
And should we write the minified code if there are errors?

Reply:
For Tracing sure.
I don't know, I don't know what NUglify does if there are errors. Probably we should write unminified if there are errors?

Reply:
Based on [this](https://github.com/xoofx/NUglify/blob/master/src/NUglify/UglifyError.cs#L133), I wonder if we should treat any error as an exception and interrupting the process...
What do you think?

Reply:
Is minification only used in production builds? If so I don't think it would be a problem. But if it's used in `taste` wouldn't that blow up the watcher?

Reply:
Well, after some search I think that it is not used at all since it is no referenced by another class and it does not implement `ITransform`...
So, for now, let's say that a reporting through `Tracing.Error` will be enough.

Reply:
Okay, i'll fix this tomorrow